src/test/java/org/osiam/client/OsiamConnectorTest.java

```diff
@@ -0,0 +1,24 @@
+package org.osiam.client;
+
+import org.junit.Test;
+
+public class OsiamConnectorTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void throws_illegal_state_exception_when_no_resource_server_is_configured_and_user_is_retrieved() {
+        OsiamConnector connector = new OsiamConnector.Builder().setAuthServerEndpoint("irrelevant").build();
+        connector.getUser("irrelevant", null);
```

Comment:
Using the literal `String` makes it instantly obvious what value is used. I think that understandability (I made that word up) weighs more than having non-duplicated literals.